contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -317,6 +317,19 @@ check_identity_insert(char** newval, void **extra, GucSource source)
 	return true;
 }
 
+static void
+throw_error_for_identity_insert(char *database, char *schema, char* object)
+{
+	ereport(ERROR,
+			(errcode(ERRCODE_INVALID_PARAMETER_VALUE),
+			 errmsg("Cannot find the object \"%s%s%s%s%s\" because it does not exist or you do not have permissions.",
+				database ? database : "",
+				database ? "." : "",
+				schema ? schema : "",
+				schema ? "." : "",
```

Comment:
It will produce db1.obj1 instead of db1..obj1 when there is no schema name supplied.

Reply:
Actually `SET IDENTITY_INSERT db1..obj1 ON/OFF` will throw `syntax error at or near ".."`